packages/runtime-core/src/errorHandling.ts

```diff
@@ -113,7 +113,9 @@ export function handleError(
       const errorCapturedHooks = cur.ec
       if (errorCapturedHooks) {
         for (let i = 0; i < errorCapturedHooks.length; i++) {
-          if (errorCapturedHooks[i](err, exposedInstance, errorInfo)) {
+          if (
+            errorCapturedHooks[i](err, exposedInstance, errorInfo) === false
```

Comment:
`!errorCapturedHooks[i](err, exposedInstance, errorInfo)` is better.

Reply:
false !== falsy values, and vue 2 uses `=== false`